srcalc/src/main/webapp/css/srcalc.css

```diff
@@ -99,6 +99,31 @@ a.btn-link:link, a.btn-link:visited {
 	text-decoration: none;
 }
 
+a.btn-em:link {
+	background: #CCCCCC;
+	font-size: 1em;
+}
+
+/* A button that is an emphasized version of the normal button.
+*/
+button.button-em {
+    
+    display: inline-block;
+	color: #666666;
```

Comment:
Two thoughts here:
* For some reason, the button-em is actually a little shorter (on Firefox and IE) than the normal buttons.
* Why a gray color for the text? This makes the button look disabled, even though we are prompting users to click it!